src/main/webapp/WEB-INF/tags/module/purap/camsDetail.tag

```diff
@@ -234,8 +234,7 @@
                 <purap:camsLocation camsLocationAttributes="${camsLocationAttributes}" ctr="${ctr}"
                                     ctr2="new"
                                     camsAssetLocationProperty="${locationPrefix}newPurchasingCapitalAssetLocationLine"
-                                    availability="${availability}" poItemInactive="${poItemInactive}"
-                                    fullEntryMode="${fullEntryMode}"/>
+                                    availability="${availability}" poItemInactive="${poItemInactive}"/>

```

Comment:
I don't recall this change being part of the original PR for this feature on the KFS 5.3.1 line. Was this supposed to be for a different PR?